pkgs/by-name/fi/finamp/package.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, flutter
+, mpv
+, xdg-user-dirs
+, patchelf
+, fetchFromGitHub
+, makeDesktopItem
+}:
+let
+  version = "0.9.8-beta";
+in
+flutter.buildFlutterApplication {
+  inherit version;
+  pname = "finamp";
+  src = fetchFromGitHub {
+    owner = "jmshrv";
+    repo = "finamp";
+    rev = version;
+    hash = "sha256-lvjhA+hdCXgDsrNhNw4Tiq6ZgkYlPuMeHha8OJNF1TI=";
+  };
+  pubspecLock = lib.importJSON ./pubspec.lock.json;
+
+  nativeBuildInputs = [ patchelf ];
+  buildInputs = [ mpv xdg-user-dirs ];
```

Comment:
`xdg-user-dirs` also needs to be a run-time dependency.

Reply:
I added it to `PATH` using `extraWrapProgramArgs` and removed it from `buildInputs`.

Reply:
Ok great. Works perfectly now @DSeeLP 
Good work!